phpBB/assets/javascript/core.js

```diff
@@ -551,11 +538,11 @@ phpbb.search.setValueOnClick = function($input, value, $row, $container) {
  * already, those are displayed instead. Executes the AJAX request function
  * itself due to the need to use a timeout to limit the number of requests.
  *
- * @param array data			Data to be sent to the server.
- * @param object event			Onkeyup event object.
- * @param function sendRequest	Function to execute AJAX request.
+ * @param {Array} data				Data to be sent to the server.
```

Comment:
Array isn't a type, it's a constructor, so yes, it should be capital.

![screen shot 2015-05-29 at 17 47 07](https://cloud.githubusercontent.com/assets/472830/7886500/be0ab592-062a-11e5-8f16-cdd9b1755406.png)